Mathlib/RingTheory/MvPowerSeries/Evaluation.lean

```diff
@@ -60,6 +60,7 @@ variable {φ : R →+* S}
 open WithPiTopology
 
 /-- Families at which power series can be consistently evaluated -/
+@[mk_iff hasEval_def]
```

Comment:
I presume that this builds `hasEval_def` which allows to unfold `hasEval` and leaves the conjunction of the two properties.

Reply:
Exactly.